worlds/animal_well/names.py

```diff
@@ -463,3 +464,120 @@ def __str__(self) -> str:
     k_medal = "Assembled the K. Medal"
     kangaroo_first_spot = "Kangaroo First Spot"  # first spot the kangaroo appears
     victory_first = "First Victory"
+
+    # fruits, by row and column, top left room is R1C1
+    fruit_0 = "Big Blue Fruit Before Disc Spike Bunny - R1C1a"  # big blue fruit, disc spike room
+    fruit_1 = "Big Blue Fruit After Disc Spike Bunny - R1C1b"  # big blue fruit, disc spike room
+    fruit_2 = "Pink Fruit by Brown Egg Chest - R1C3"
+    fruit_3 = "Blue Fruit Above Dog Fast Travel - R1C4"  # blue fruit
+    fruit_4 = "Blue Fruit by Raw Egg Puzzle Entrance - R1C6a"  # blue fruit
+    fruit_5 = "Pink Fruit Below Raw Egg Puzzle Entrance - R1C6b"
+    fruit_6 = "Pink Fruit Left of Mock Disc Shrine - R1C7"
+    fruit_7 = "Blue Fruit Top of Dog Elevator - R1C9"  # blue fruit
+    fruit_8 = "Pink Fruit in Bear Upper Phone Room Left - R1C11a"
+    fruit_9 = "Pink Fruit in Bear Upper Phone Room Right - R1C11b"
+    fruit_10 = "Pink Fruit Right of Ferrets - R1C15"
+    fruit_11 = "Pink Fruit by Swordfish Lake - R2C3"
+    fruit_12 = "Pink Fruit by Dog Fast Travel - R2C4"
+    fruit_13 = "Blue Fruit by P. Flame - R2C9"  # blue fruit
+    fruit_14 = "Pink Fruit in Dog Elevator by P. Flame - R2C10"
+    fruit_15 = "Blue Fruit by V. Flame - R2C11"  # blue fruit
+    fruit_16 = "Pink Fruit in Second Chameleon Fight - R2C12"
+    fruit_17 = "Pink Fruit in Bat Room - R3C3"
+    fruit_18 = "Pink Fruit by Barcode Bunny - R3C5"
+    fruit_19 = "Blue Fruit in Dog Switch Maze - R3C7"  # blue fruit
+    fruit_20 = "Pink Fruit Right of Dog Switch Maze Left - R3C8a"
+    fruit_21 = "Pink Fruit Right of Dog Switch Maze Right - R3C8b"
+    fruit_22 = "Pink Fruit in Dog Elevator Below P. Flame - R3C10"
+    fruit_23 = "Pink Fruit in Bear Middle Phone Room - R3C12"
+    fruit_24 = "Pink Fruit by Monkey - R3C16"
+    fruit_25 = "Blue Fruit in Kangaroo Room Hand - R4C1a"  # blue fruit upper
+    fruit_26 = "Blue Fruit in Kangaroo Room Tail - R4C1b"  # blue fruit lower
+    fruit_27 = "Pink Fruit in Kangaroo Room Foot - R4C1c"
+    fruit_28 = "Pink Fruit by Turtle Flute Pool - R4C6"
```

Comment:
```suggestion
    fruit_28 = "Pink Fruit by Turtle Pool - R4C6"
```
Small change for conciseness.
I do really like calling this "turtle pool". In other parts of the logic, we've just referred to this as "lake", as in dog_upper_past_lake, which is already kinda confusing and redundant with swordfish lake. Maybe there's room for us to change up the name in other parts to match.